code/track.ss

```diff
@@ -9,19 +9,9 @@
       (delete-file config.json))
     (with-output-to-file config.json
       (lambda ()
-	(json-write (process-config))))))
-
-;; preprocess nodes in config. currently removes exercises marked wip
-(define (process-config)
```

Comment:
This has been moved to `outils.ss`

Reply:
I think it should stay in `track` since it deals specifically with the config. `outils` is intended for generally useful helpers. `process-config` is specific to selecting which problems should be used.

Reply:
Okay, that is fine with me.  I think at some point we should actually just move all of the config helpers into config-utils.ss.

Do you want me to go ahead and do that, or just move `preprocess-config` back to `track.ss`?

Reply:
Hmm, the helpers I made for `process-config` are currently only useful to the config stuff, but they are more general than the config stuff.

They seem like they are really formatting utilities to help with compatibility with the web site.

What do you think about storing them in a file called `compatibility-utils.ss`?

Then `process-config` would just go back into `track.ss`.

Reply:
I know I would have to modify `load.ss` then, so if you think that kind of refactoring should just happen in another PR, then I will go ahead with the simple suggestion you gave.

Reply:
I'm not sure I understand what `compatibility` means here. I think it does sound like these sorts of changes would be better from another pr. The json pretty-printing functionality shouldn't really be leaking from `json.sls`. Granted, it's specific to making exercism/configlet happy, so perhaps a separate file dedicated to it makes the most sense. It's not general enough to be in `json.sls`, and it's internals should be sealed somewhat.

Reply:
Yeah, so I wasn't meaning to have anything from `json.sls` leak into another file.

Basically the idea is just that the helpers that convert parts of the config to snake case (since some of them are general even though they only have one application now) could be put in a file that has a name concerning compatibility.  If everything were scheme all the time we wouldn't have to worry about it.

I'll go with your first suggestion for now ;)

Reply:
Sounds good. Snake case conversion stuff could go in `outils.ss` though

Reply:
Yeah, I kept it there at least for this PR.